src/pal/src/exception/seh-unwind.cpp

```diff
@@ -251,6 +254,11 @@ BOOL PAL_VirtualUnwind(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *contextP
         return FALSE;
     }
 #endif
+
+#if !UNWIND_CONTEXT_IS_UCONTEXT_T
+    UpdateUnwindContextWithWinContext(context, &unwContext);
+#endif
+
```

Comment:
Do  you mean that UNWIND_CONTEXT_IS_UCONTEXT_T = 1 always holds for I386 / AMD64 / ARM64? If so, we can eliminate the call of ``WinContextToUnwindCursor`` at line 260:262.